mParticle-Rokt/MPKitRokt.m

```diff
@@ -52,6 +61,29 @@ - (void)start {
 
     dispatch_once(&kitPredicate, ^{
         self->_started = YES;
+        
+        NSDictionary *attributes = @{
+                @"email" : @"j.smith@example.com",
+                @"firstname": @"Jenny",
+                @"lastname": @"Smith",
+                @"mobile": @"(555)867-5309",
+                @"postcode": @"90210",
```

Comment:
If you want to use this attribute for testing, could you please send the `@"sandbox" : @"true",` as well?
https://docs.rokt.com/developers/integration-guides/ios/how-to/sandbox